internal/provider/provider.go

```diff
@@ -64,8 +68,87 @@ func New() func() *schema.Provider {
 	}
 }
 
+const statuspageUrl = "https://pdrzb3d64wsj.statuspage.io/api/v2/components.json"
```

Comment:
Will this always check the production status? Maybe an edge case, but it seems like this could unnecessarily disrupt testing in lower envs.

Reply:
Excellent point! Went ahead and added a HCP_HOST_PATH check so this only runs when pointed at prod.